features/conference/components/Conference.js

```diff
@@ -51,7 +68,15 @@ class Conference extends Component {
      * @returns {void}
      */
     _onPress() {
-        this.setState({ isToolbarVisible: !this.state.isToolbarVisible });
+        var isVisible = !this.state.isToolbarVisible;
+
+        this.setState({ isToolbarVisible: isVisible });
+
+        if (isVisible)
+            this.toolbarTimeout = setTimeout(
+                this._onPress,
+                toolbarTimeoutValue
+            );
```

Comment:
This logic if flawed because it doesn't properly clear the toolbarTimeout. It effectively allows a sequence of multiple fast taps to schedule an equal number of timeouts which may result in user hiding the toolbar & then a timeout showing it & then another timeout hiding it again & so on & so forth. Indeed, that's easy to reproduce: start with the toolbar hidden, tap twice in a row in less than 5 seconds (which should result in the toolbar being hidden) but spead them by at least 1 second so that you can observe the 2 timeouts firing later on & then wait until the toolbar appears, then wait for it to disappear again.